.github/actions/install-nixie/tests/test_action.py

```
@@ -0,0 +1,152 @@
+""",
+        )
+
+    env = {
+        **os.environ,
+        "CALLS_PATH": calls_path.as_posix(),
+        "MERMAN_VERSION": "0.7.0",
+        "NIXIE_VERSION": "1.1.0",
+        "PATH": stubs_dir.as_posix(),
+        "PYTHON_VERSION": "3.14",
+    }
+    return subprocess.run(  # noqa: S603,TID251 - exercise the bash fragment.
```

Ruff (0.15.21)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)